web/pdf_link_service.js

```diff
@@ -146,35 +96,39 @@ class PDFLinkService {
    * @type {number}
    */
   get page() {
-    return this.pdfViewer.currentPageNumber;
+    return this.pdfDocument ? this.pdfViewer.currentPageNumber : 1;
```

Comment:
*Please note:* This "new" fallback value is consistent with the initial value used in the viewer, see https://github.com/mozilla/pdf.js/blob/90d4b9c2c0df7cadbd5c2388431b208f08da1ed6/web/pdf_viewer.js#L1064-L1066